README.md

```diff
@@ -106,25 +107,35 @@ name
 height
 ```
 
-Entity names and attributes may **not** contain spaces.
+Entity names and attributes may contain spaces and mostly any character,
+except ASCII control characters like carriage return and line feed,
+if quoted with backticks, simple quotes or double quotes:
 
-Any number of attributes may be declared as a primary key for its entity by 
-prefixing the attribute with a `*`. Similarly, an attribute may be declared
+```
+[`Birth Place`]
+*id
+`birth city`
+'birth state'
+"birth country"
+```
+
+Any number of attributes may be declared as a primary key for its entity by
+prefixing the attribute with a `\*`. Similarly, an attribute may be declared
```

Comment:
Why was `*` changed to `\*`? Doesn't seem correct to me. Same change was done at line 133.

Reply:
I have to confess that I had no recollection 4 years later, but I found a hint in the title of the corresponding commit, e8d865603: `Escape * as \* to avoid bold formatting in syntax highlighting`. This is simply Markdown formatting. Whether it is the correct interpretation of Markdown format or not, I cannot say, but the unescaped `*` was interpreted as the start of a span of bold text in my editor (`vim`), which was really annoying.

Reply:
I hadn't noticed some time went between the creation and merging of this PR 😛 Likely it was a problem with your editor, no escaping should be needed because it's an *inline code* section. I'll do a PR to fix it.